.env

```diff
@@ -0,0 +1,5 @@
+STORAGE_API_URL=
+TEST_MANAGE_API_APPLICATION_TOKEN=
+TEST_NOTIFICATION_API_URL=

```

Comment:
Proc `STORAGE_API_URL` vs `TEST_NOTIFICATION_API_URL` (`TEST_` prefix)?

Reply:
nevím, bylo to tak :)